src/components/Dog.jsx

```diff
@@ -1,16 +1,33 @@
-import PropTypes from "prop-types";
-import "./Dog.css";
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import './Dog.css';
 
 const Dog = (props) => {
+    const [chipNum, setChipNum] = useState(props.chip);
```

Comment:
This has the possibility of reinforcing an anti-pattern. Generally, we don't pass props and then turn around and store it in state. I recognize that it's like that here because we haven't yet lifted up state, and this is sort of standing in for an edit form, but I'd like to at least see a comment to explain to student that this is not the way to think about the relationship between props and state. That is, we don't usually "cache" the prop in state, since doing so would cause the component to ignore subsequent updates to props (since the props version is only used to initialze the state).

The curriculum dodges the issue by not passing the toggle-able value as props at all (defaults to false in the state) until we lift the state up.